samples/bluetooth/channel_sounding_ras_reflector/src/main.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *  @brief Channel Sounding Reflector with Ranging Responder sample
+ */
+
+#include <zephyr/sys/printk.h>
+#include <zephyr/types.h>
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/cs.h>
+#include <bluetooth/services/ras.h>
+
+#include <dk_buttons_and_leds.h>
+
+#define CON_STATUS_LED DK_LED1
+
+static K_SEM_DEFINE(sem_connected, 0, 1);
+
+static struct bt_conn *connection;
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_RANGING_SERVICE_VAL)),
+	BT_DATA(BT_DATA_NAME_COMPLETE, CONFIG_BT_DEVICE_NAME, sizeof(CONFIG_BT_DEVICE_NAME) - 1),
+};
+
+static void connected_cb(struct bt_conn *conn, uint8_t err)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	(void)bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+	printk("Connected to %s (err 0x%02X)\n", addr, err);
+
+	if (err) {
+		bt_conn_unref(conn);
+		connection = NULL;
+	}
+
+	connection = bt_conn_ref(conn);
+
+	k_sem_give(&sem_connected);
+
+	dk_set_led_on(CON_STATUS_LED);
+}
+
+static void disconnected_cb(struct bt_conn *conn, uint8_t reason)
+{
+	printk("Disconnected (reason 0x%02X)\n", reason);
+
+	bt_conn_unref(conn);
+	connection = NULL;
+
+	dk_set_led_off(CON_STATUS_LED);
+}
+
+static void remote_capabilities_cb(struct bt_conn *conn, struct bt_conn_le_cs_capabilities *params)
+{
+	ARG_UNUSED(conn);
+	ARG_UNUSED(params);
+	printk("CS capability exchange completed.\n");
+}
+
+static void config_created_cb(struct bt_conn *conn, struct bt_conn_le_cs_config *config)
+{
+	ARG_UNUSED(conn);
+	printk("CS config creation complete. ID: %d\n", config->id);
+}
+
+static void security_enabled_cb(struct bt_conn *conn)
+{
+	ARG_UNUSED(conn);
+	printk("CS security enabled.\n");
+}
+
+static void procedure_enabled_cb(struct bt_conn *conn,
+				 struct bt_conn_le_cs_procedure_enable_complete *params)
+{
+	ARG_UNUSED(conn);
+	if (params->state == 1) {
+		printk("CS procedures enabled.\n");
+	} else {
+		printk("CS procedures disabled.\n");
+	}
+}
+
+BT_CONN_CB_DEFINE(conn_cb) = {
+	.connected = connected_cb,
+	.disconnected = disconnected_cb,
+	.le_cs_remote_capabilities_available = remote_capabilities_cb,
+	.le_cs_config_created = config_created_cb,
+	.le_cs_security_enabled = security_enabled_cb,
+	.le_cs_procedure_enabled = procedure_enabled_cb,
+};
+
+int main(void)
+{
+	int err;
+
+	printk("Starting Channel Sounding Reflector Sample\n");
```

Comment:
replace prink with LOG_* calls instead

Reply:
I think by default LOG_* and printk() use the same backend anyway. (CONFIG_LOG selects CONFIG_PRINTK => makes CONFIG_LOG_PRINTK default to y).

There's also many existing samples that use printk()

Reply:
Updated to use logging now but I agree, would be nice if all the samples were consistent in their logger usage